### DISCUSSION TITLE
Paste to upload no longer working

### DISCUSSION THREAD

**Discussion by tanookiben** [LycheeOrg/Lychee#2167] [Bugs] [answered]
I'm guessing this may be related to v5 but I don't see anything in the [release notes](https://github.com/LycheeOrg/Lychee/releases/tag/v5.0.0) related to this

Previously I was able to upload images (like a screenshot) by pasting with CTRL+V on Windows/Chrome on the main gallery page, but this behavior seems to have been removed. Was this an intentional change?

**Comment by ildyria** [comment]
This was not an intentional change for sure.

One of the change I haven't had time to add back was the drag&drop on window when the upload modal is not shown.
But I didn't know that the ctrl+v was even a thing. Can you try to do `U` (to make the upload modal pop) and then `CTRL + V`, see if this works (by some miracle) ?

**Reply by tanookiben** [reply]
Nope, unfortunately `CTRL+V` on the upload modal also doesn't work in v5

**Reply by kamil4** [reply]
@ildyria FYI: https://github.com/LycheeOrg/Lychee-front/pull/153

**Reply by ildyria** [reply]
oh, thanks @kamil4

**Comment by ildyria** [comment, marked as answer]
Fixed in v6. :)
